authentication.js

```diff
@@ -161,7 +161,7 @@ module.directive('dbUserLogin', ['Session', 'SessionStore' , function(Session, S
       organization: "=dbOrganization"
     },
     link: function(scope, element, attributes, ngModel) {
-      scope.user = {};
+      scope.user = {organization_id: "-1"};
```

Comment:
I specifically do not want this handled here. It should be handled in the API as it was before.